src/ScrollSync.js

```diff
@@ -109,18 +119,35 @@ export default class ScrollSync extends Component {
     const scrollTopOffset = scrollHeight - clientHeight
     const scrollLeftOffset = scrollWidth - clientWidth
 
-    const { proportional, vertical, horizontal } = this.props
+    const {
+      proportional,
+      vertical,
+      horizontal,
+      initialScrollTop,
+      initialScrollLeft
+    } = this.props
 
     /* Calculate the actual pane height */
     const paneHeight = pane.scrollHeight - clientHeight
     const paneWidth = pane.scrollWidth - clientWidth
     /* Adjust the scrollTop position of it accordingly */
     if (vertical && scrollTopOffset > 0) {
-      pane.scrollTop = proportional ? (paneHeight * scrollTop) / scrollTopOffset : scrollTop // eslint-disable-line
+      if (!this.state.initialized) pane.scrollTop = initialScrollTop
+      // eslint-disable-next-line curly
+      else
+        pane.scrollTop = proportional
+          ? (paneHeight * scrollTop) / scrollTopOffset
+          : scrollTop // eslint-disable-line
     }
     if (horizontal && scrollLeftOffset > 0) {
-      pane.scrollLeft = proportional ? (paneWidth * scrollLeft) / scrollLeftOffset : scrollLeft // eslint-disable-line
+      if (!this.state.initialized) pane.scrollLeft = initialScrollLeft
+      // eslint-disable-next-line curly
+      else
+        pane.scrollLeft = proportional
+          ? (paneWidth * scrollLeft) / scrollLeftOffset
+          : scrollLeft // eslint-disable-line
     }
+    if (!this.state.initialized) this.state.initialized = true
```

Comment:
I think React state should not be mutated like this. Either use `setState` or don't use `state` for it. Maybe that's there the ESLint error comes from?